packages/ListBox/README.md

```diff
@@ -0,0 +1,80 @@
+## Date picker
+
+### Installation
+
+`> npm install --save @paprika/listbox`
+or
+`> yarn add @paprika/listbox`
+
+### Usage
+
+For a basic ListBox
+
+```js
+import ListBox from "@paprika/listbox";
+
+function Component() {
+  return (
+    <ListBox onChange={changeHandler}>
+      <ListBox.Option>Option 1</ListBox.Option>
+      <ListBox.Option>Option 2</ListBox.Option>
+    </ListBox>
+  );
+}
+```
+
+With Filter
+
+```js
+import ListBox from "@paprika/listbox";
+
+function Component() {
+  return (
+    <ListBox onChange={changeHandler}>
+      <ListBox.Filter />
+      <ListBox.Option>Option 1</ListBox.Option>
+      <ListBox.Option>Option 2</ListBox.Option>
+    </ListBox>
+  );
+}
+```
+
+### Props
+
+```js
+/** Child of type <ListBox.Option /> */
+children: PropTypes.node,
```

Comment:
Do we really want to duplicate the amount of maintenance we'll have to do when we update this component's props by copying these into the `README.md`?

Reply:
good point! those shouldn't be here, I guess a prop table should care about this! I will remove it.